### ISSUE TITLE
Get many warning when running with Crystal 1.11.2

### ISSUE THREAD

**Issue by zw963** [Sija/debug.cr#10]
```
lib/debug/core_ext/kernel.cr:7:25

 7 | ::Debug.log({{ *args }})

Warning: Deprecated use of splat operator. Use `#splat` instead
```

For now, have to use debug.cr like this to prohibit those anoise debug info.

```
crystal run --warnings none 1.cr
```

Thanks

**Comment by Sija** [comment]
Fixed in 6939d5d264645118c44bc1b12f6e83005f72d56c & a45e2e9e71bfa1417af39ff69d77854188cb9ccc

**Comment by Sija** [comment]
Released v2.0.3 with these changes.
